merino-web/src/endpoints/suggest.rs

```diff
@@ -187,6 +193,14 @@ fn safe_log_request(
                 .join(",")
         },
     );
+    let mut seq = "".to_string();
+    if let Some(sequence) = query_params.sequence {
+        seq = sequence.to_string();
+    }
+    let mut sid = "".to_string();
+    if let Some(session_id) = query_params.session_id {
+        sid = session_id.to_string();
+    }
```

Comment:
gut feeling says this is kinda jank, but my limited rust experience couldn't figure out how to make it less jank :(